workloads/hyper-scale/common.sh

```diff
@@ -2,12 +2,13 @@
 set -x
 
 source env.sh
+TEMP_DIR=`mktemp -d`
 
 prep(){
     if [[ -z $(go version) ]]; then
         curl -L https://go.dev/dl/go1.18.2.linux-amd64.tar.gz -o go1.18.2.linux-amd64.tar.gz
-        tar -C /usr/local -xzf go1.18.2.linux-amd64.tar.gz
-        export PATH=/usr/local/go/bin:$PATH
+        tar -C ${TEMP_DIR}/ -xzf go1.18.2.linux-amd64.tar.gz
+        export PATH=${TEMP_DIR}/go/bin/:$PATH
```

Comment:
Don't think we need the trailing / on ${TEMP_DIR}/go/bin/

Reply:
Done!